### PR TITLE
[MIG][14.0] hr_presence: migration script

### PR DESCRIPTION
**Pull request by tranngocson1996**
This PR
-------
Migration done

### CODE REVIEW

File: openupgrade_scripts/scripts/hr_presence/14.0.1.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(
+        env.cr,
+        "hr_presence",
+        "14.0.1.0/noupdate_changes.xml",
+    )
```
Comment (MiquelRForgeFlow) [inline openupgrade_scripts/scripts/hr_presence/14.0.1.0/post-migration.py:10]:
```suggestion
    )
    delete_record_translations(env.cr, "hr_presence", ["mail_template_presence", "sms_template_data_hr_presence"])
```

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by tranngocson1996** [comment]
@MiquelRForgeFlow done

File: openupgrade_scripts/scripts/hr_presence/14.0.1.0/post-migration.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply openupgrade_scripts/scripts/hr_presence/14.0.1.0/post-migration.py:10]: > openupgrade.delete_record_translations

Comment (tranngocson1996) [reply openupgrade_scripts/scripts/hr_presence/14.0.1.0/post-migration.py:10]: @MiquelRForgeFlow sr typo error

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**